homeassistant/components/caldav/calendar.py

```diff
@@ -151,6 +141,7 @@ def update(self):
         event = calculate_offset(event, OFFSET)
         self._offset_reached = is_offset_reached(event)
         self._event = event
+        self._attr_extra_state_attributes = {"offset_reached": self._offset_reached}
```

Comment:
`self._offset_reached` isn't used now anymore, and can be cleaned up.

```suggestion
        self._attr_extra_state_attributes = {"offset_reached": is_offset_reached(event)}
```

Reply:
Wasn't sure if I could remove it. Thanks